testproject/tests/test_second_step_authentication.py

```diff
@@ -507,28 +462,28 @@ def test_change_primary_method_to_invalid_wrong(active_user_with_many_otp_method
     response = client.post(
         path="/auth/mfa/change-primary-method/",
         data={
-            "method": "sms",
+            "method": "sms_twilio",
             "code": "test",
         },
         format="json",
     )
-    error_code = "invalid_code"
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too